setup.py

```diff
@@ -20,7 +20,7 @@
     zip_safe=False,
     entry_points={
         'console_scripts': [
-            'ansible-runner=ansible_runner.interface:main',
-        ],
-    },
+            'ansible-runner = ansible_runner.__main__:main'
```

Comment:
:+1: I like the rename to `__main__` for the module and that follows best practice.